tests/manual_testvectors.json

```diff
@@ -53,28 +53,31 @@
         "kind": "RegisterEntity",
         "signature_context": "oasis-core/consensus: tx for chain test-chain-id",
         "tx": {
+            "fee": {
+                "gas": 0,
+                "amount": "0"
+            },
             "body": {
                 "signature": {
                     "signature": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==",
                     "public_key": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
                 },
-                "untrusted_raw_value": {
-                    "entity": {
-                        "id": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASM=",
-                        "nodes": [
-                            "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAg=",
-                            "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAk="
-                        ],
-                        "allow_entity_signed_nodes": true
-                    }
-                }
+                "untrusted_raw_value": "a362696458200000000000000000000000000000000000000000000000000000000000000000656e6f6465738258200000000000000000000000000000000000000000000000000000000000000001582000000000000000000000000000000000000000000000000000000000000000027819616c6c6f775f656e746974795f7369676e65645f6e6f646573f5"
             },
             "nonce": 123,
             "method": "registry.RegisterEntity"
         },
-        "encoded_tx": "o2JpZFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASNlbm9kZXOCWCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAl4GWFsbG93X2VudGl0eV9zaWduZWRfbm9kZXP1",
+        "entity": {
```

Comment:
mmm.. I don't think Oasis test vectors will look like this... actually entity is supposed to come inside of untrusted_raw_value. 
The previous test generation script was doing that correctly, though, there might be a bug somewhere else.
I will review/debug a bit more the JS script but I prefer not to make this change. Hopefully, complete test vectors for all these new types will be available soon